infrastructure-playbooks/rolling_update.yml

```diff
@@ -76,15 +79,11 @@
   become: True
 
   pre_tasks:
-    - include_vars: roles/ceph-common/defaults/main.yml
```

Comment:
That appears to only be a problem for relative paths without `role` as the directory name. It may be a problem if we're running the `rolling_update.yml` playbook from within `infrastructure-playbooks` as I have not tried that recently, but I thought your most recent suggestion via this [commit](https://github.com/ceph/ceph-ansible/blob/b0989c700f9a4651b2cae978653d255ef0e0b4d9/infrastructure-playbooks/README.md) was to copy them into the root directory and run it from there.